src/components/Input/styled/iconContainer.js

```diff
@@ -1,12 +1,16 @@
 import styled from 'styled-components';
-import { COLOR_GRAY_3 } from '../../../styles/colors';
 import getTheme from '../../../styles/helpers/getTheme';
 
 const IconContainer = styled.span.attrs(props => {
-    const errorMainColor = getTheme(props).palette.error.main;
-    return { errorMainColor };
+    const palette = getTheme(props).palette;
+    const errorMainColor = palette.error.main;
+    const headerText = palette.text.header;
+    return {
+        errorMainColor,
+        headerText,
+    };
 })`
-    fill: ${COLOR_GRAY_3};
+    color: ${props => props.headerText};
```

Comment:
@TahimiLeonBravo check this maybe we can define color for icons or other since header text color is not semantic to use in icon fill color